05/app.js

```diff
@@ -1,53 +1,82 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+	paragraphs: {
+		p1: 0,
+	},
+	links: {
+		"/dolor.html": 0,
+	},
 };
 
 /* tutaj umieść swój kod */
 
+const pList = document.querySelectorAll("p");
 
-/* nie modyfikuj kodu poniżej, ale przeanalizuj go */
+pList.forEach(function (item) {
+	item.addEventListener("click", function (e) {
+		if (e.target === e.currentTarget) {
+			let itemId = item.dataset.id;
 
-const statsElement = document.querySelector('.stats');
-const fireCustomEvent = function(element, name) {
-    console.log(element, '=>', name);
+			if (typeof stats.paragraphs[itemId] === "undefined") {
+				stats.paragraphs[itemId] = 1;
+			} else {
+				stats.paragraphs[itemId] += 1;
+			}
+		} else if (e.target !== e.currentTarget) {
```

Comment:
Zwróć uwagę, że wyżej już masz warunek odpowiedni tj. `e.target === e.currentTarget` skoro jest on niespełniony to wiadomo że `e.target !== e.currentTarget` dlatego nie ma sensu pisać `else if`, wystarczy `else` :)